token_reset/example.txt

```diff
@@ -1 +0,0 @@
-cf-autotick-bot-test-package
```

Comment:
Looks like `token_reset/example.txt` was accidentally deleted here. Readding it in PR ( https://github.com/conda-forge/admin-requests/pull/611 )